conda/meta.yaml

```diff
@@ -62,6 +63,7 @@ requirements:
     - r-tidyr
     - r-stringr
     - r-tibble
+    - r-microbenchmark
```

Comment:
```suggestion
```

This is not required in the meta.yaml file

Reply:
done in [5c6a200](https://github.com/RECETOX/recetox-aplcms/pull/131/commits/5c6a20041696c00bc1f3761f3d66f2c092a64aff)